packml_msgs/msg/Stats.msg

```diff
@@ -0,0 +1,29 @@
+# This message contains machine agnostic information that is extracted from PackML
+# standard states
+
+std_msgs/Header stamp       # Duration of statistics calculation
+std_msgs/Duration duration      # Duration over which statistics are calculated
```

Comment:
I think in this case, we should enable more resolution where possible.  I don't believe the messages should match the standard 1 for 1 (this is up for debate).  There will need to be a shim layer/library that converts to the PackML standard.